components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/impl/ProvisioningClient.java

```diff
@@ -355,6 +360,102 @@ public void provisionUpdateUser() {
 
     }
 
+    public void provisionPatchUser() {
+        try {
+            //get provider details
+            String userEPURL = provider.getProperty(SCIMConfigConstants.ELEMENT_NAME_USER_ENDPOINT);
+            String userName = provider.getProperty(SCIMConfigConstants.ELEMENT_NAME_USERNAME);
+            String password = provider.getProperty(SCIMConfigConstants.ELEMENT_NAME_PASSWORD);
+            String contentType = provider.getProperty(SCIMConstants.CONTENT_TYPE_HEADER);
+
+            if (contentType == null) {
```

Comment:
Since apache.commons is already used, can you use StringUtils.isEmpty(contentType) so that it will check for both null and empty string.